### PR TITLE
[kinetic] use industrial_ci for travis

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,27 @@
-sudo: required
+udo: required
```
Comment (NadiaHG) [inline .travis.yml]: **s**udo

**Review by NadiaHG — DISMISSED** [review]

**Comment by benmaidel** [comment]
@ipa-nhg please review or approve

**Comment by benmaidel** [comment]
the never ending story of dependencies ;)

**Review by fmessmer — APPROVED** [review]
see https://github.com/ipa320/cob_robots/pull/559#issuecomment-276625086

but alphabetic order in .rosinstall would be nice!

**Comment by fmessmer** [comment]
I will merge all [kinetic]-tagged PRs before cleaning up branches...

**Pull request merged**